.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Rust nightly
+        run: rustup toolchain install nightly
+
+      - name: Run rustfmt (nightly)
+        run: cargo +nightly fmt --all -- --config format_code_in_doc_comments=true
```

Comment:
Thank you @yazanmashal03  

I am not sure what this check is doing - it seems to just run the fmt rather than check its output, for example